nisystemlink/clients/core/_internal/_winpaths.pyi

```diff
@@ -4,11 +4,9 @@ import ctypes
 from ctypes import wintypes
 from uuid import UUID
 
-
```

Comment:
I'm curious about tehse edits come from, and for stubs/events.pyi as well. Aren't there usually two blank lines between classes? Also, it looks as though the black command is intended to exclude pyi files.

Reply:
These changes did come from black, (running `poe format`) - so it looks like there might be an issue with the regex that is supposed to exclude pyi files

Reply:
Looks like there was a Windows-specific issue when specifying the `--exclude` regex inline in the poe task. Moving the exclude config out to a black-specific section in the pyproject.toml file seems to have fixed it.